app/src/main/java/com/kickstarter/ui/viewholders/CreatorDashboardHeaderViewHolder.java

```diff
@@ -90,14 +102,18 @@ public CreatorDashboardHeaderViewHolder(final @NonNull View view) {
       .subscribe(projectAndRefTag -> this.startProjectActivity(projectAndRefTag.first, projectAndRefTag.second));
   }
 
+  @OnClick(R.id.project_name_blurb_arrow_view)
+  protected void dashboardShowProjectMenuClicked() {
+    this.delegate.projectsMenuClicked();
```

Comment:
wanna mention that I removed passing `this` here because it seemed like it was easier to set up tests if we aren't passing the ViewHolder.  I know we chatted about this irl though so wanted to point it out.

Reply:
sounds good to me!